### ISSUE TITLE
[BUG] CANT LOG INTO STREAMLABS OBS

### ISSUE THREAD

**Issue by fireshot206** [rivafarabi/deckboard#562]
Downloaded the app on both PC and tablet and works great. I'm able to make my buttons work as they should. For streaming purposes and controlling my scenes from my tablet, it says I need to connect to streamlabs using the icon at the top of the pc app under the start button. Unfortunately for me, whenever I click the icon and try to sign in to streamlabs, nothing pops up at all. Same thing goes for obs and spotify, no matter how many times I try to click on them, nothing opens up for me to sign in. Please help! Ive uploaded a zip file with a display capture showing what I mean.

[deckboard cant connect.zip](https://github.com/rivafarabi/deckboard/files/5542051/deckboard.cant.connect.zip)

**Comment by rivafarabi** [comment]
Have you followed the instruction on this [page ](https://github.com/rivafarabi/deckboard/wiki/Streamlabs-OBS-Setup)?

**Comment by fireshot206** [comment]
Yes, I have implemented all the settings and yet I am still unable to log in to streamlabs. The video within the zip file I provided in my previous post will show the issue I am having even when applying the settings.



> Have you followed the instruction on this [page ](https://github.com/rivafarabi/deckboard/wiki/Streamlabs-OBS-Setup)?

**Comment by MisterJack49** [comment]
I have a similar issue, I had everything configured, and even set scenes changes, but now it says that streamlabs is not configured.

I tried launching streamlabs and then deckboard, I also tried restarting deckboard without success.

**Comment by pradhanTejeshwar** [comment]
My is a similar type of error saying streamlabs OBS is not connected, even tho I have implemented the api code but doesnt give me option to log in streamlabs.
